app/components/project-settings/header/index.ts

```
@@ -12,22 +15,37 @@ interface ProjectSettingsHeaderSignature {
 
 export default class ProjectSettingsHeaderComponent extends Component<ProjectSettingsHeaderSignature> {
   @service declare intl: IntlService;
+  @service declare router: RouterService;
+  @service('breadcrumbs') declare bCS: BreadcrumbsService;
+
+  constructor(owner: unknown, args: ProjectSettingsHeaderSignature['Args']) {
+    super(owner, args);
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)